Model/Game.js

```diff
@@ -71,7 +73,8 @@ class Game {
         0,
         0.2 * screenY,
         carHeight,
-        carWidth
+        carWidth,
```

Comment:
This is what I'm talking about, you had to add a coma, so now you have to add some thirty comas, instead of just 1